llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -1726,7 +1726,7 @@ let SubtargetPredicate = isGFX12Plus in {
   defm : SIBufferAtomicPat_Common<"SIbuffer_atomic_cond_sub_u32", i32, "BUFFER_ATOMIC_COND_SUB_U32_VBUFFER", ["noret"]>;
 }
 
-let SubtargetPredicate = isGFX6GFX7GFX10Plus in {
+let OtherPredicates = [isGFX6GFX7GFX10Plus] in {
```

Comment:
It is annoying not to be able to use `SubtargetPredicate` for something that is clearly a subtarget predicate, but it clashes with the use of `SubtargetPredicate = HasUnrestrictedSOffset` inside `SIBufferAtomicPat`.

Reply:
I would expect these to be swapped, with HasUnrestrictedSOffset in OtherPredicates

Reply:
Right, but that just causes the converse problem, as there are already a bunch of cases that override `OtherPredicates` in `SIBufferAtomicPat`.